src/util/prefer-v2-deployment.ts

```diff
@@ -12,14 +12,20 @@ export async function hasDockerfile(cwd: string) {
   return new Promise(res => exists(join(cwd, 'Dockerfile'), res));
 }
 
+export async function hasServerfile(cwd: string) {
+  return new Promise(res => exists(join(cwd, 'server.js'), res));
```

Comment:
I'm not sure I understand the purpose of this PR 🤔 

What is the significance of `server.js` and  why would that change the deployment type?

Reply:
@styfle It will not change the deployment type, it just continues to stay at `1` when there is a `server.js` file, since 1.0 can use it as an entry point when there is no `start` script in the `package.json` file.

Reply:
I meant to say deployment "version", not "type" 😄 

But I'll trust you on this one since I didn't know you could do this with 1.0 🤷‍♂